pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/subjects/SierraPersonSubjectsTest.scala

```diff
@@ -339,6 +343,37 @@ class SierraPersonSubjectsTest
         "Agate, John, 1676-1720. Sermon preach'd at Exeter, on the 30th of January ...")
   }
 
+  it("trims whitespace from label-derived identifiers") {
+    // This is based on b24000802, retrieved 8 August 2022
+    val varField = VarField(
+      fieldTag = Some("d"),
+      marcTag = Some("600"),
+      indicator1 = Some(" "),
+      indicator2 = Some("0"),
+      subfields = List(
+        Subfield(
+          tag = "a",
+          content = ""
+        ),
+        Subfield(
+          tag = "a",
+          content = "Turner, John"
```

Comment:
Should this have whitespace around it?  I can't see how this test is proving that whitespace is being trimmed.

Reply:
Ah, no, the omission of whitespace is intentional – I'll add a comment to explain.